File: php/array_count9.php
<?php
	function array_count9($arr)
	{
		$count=array_count_values($arr);
		return $count[9]; 
	}
	echo array_count9(array(1,2,9))==1 ? "pass" : "fail"; echo "\n";
	echo array_count9(array(1,9,9))==2 ? "pass" : "fail"; echo "\n";
	echo array_count9(array(1,9,9,3,9))==3 ? "pass" : "fail"; echo "\n";
?>

File: ruby/array_count9.rb
def array_count9(arr)
	return arr.count(9)
end
puts array_count9([1,2,9])==1 ? "pass" : "fail"
puts array_count9([1,9,9])==2 ? "pass" : "fail"
puts array_count9([1,9,9,3,9])==3 ? "pass" : "fail"
